Patches/Sergal/PawnKinds_Lions.xml

```diff
@@ -0,0 +1,101 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationSequence">
+		<success>Always</success>
+		<operations>
+		
+			<li Class="CombatExtended.PatchOperationFindMod">
+				<modName>Sergals Gnolls Lions and Frogs</modName>
+			</li>
+			
+			<!-- Add backpacks to list of allowed apparel for Lion race -->
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/AlienRace.ThingDef_AlienRace[defName="Alien_Lion"]/alienRace/raceRestriction/whiteApparelList</xpath>
+				<value>
+					<li>Apparel_Backpack</li>
+				</value>
+			</li>
+			
+			<!-- Spawn backpacks for selected Lion pawntypes -->
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/PawnKindDef[
+					defName="LionColonist" or
+					defName="LionCommando" or
+					defName="LionCommandoElite" or
+					defName="LionHunter" or
+					defName="LionPackLeader" or
+					defName="LionRecruit" or
+					defName="LionSecurityGuard" or
+					defName="LionTrader"
+				]</xpath>
+				<value>
+					<apparelRequired>
+						<li>Apparel_Backpack</li>
+					</apparelRequired>
+				</value>
+			</li>
+			
+			<!-- Reduce max Silver carried by Military Lion pawns (original mod allowed them to have up to 20k silver coins, which weights over 100 kgs and prevents other inventory items from being spawned) -->
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/PawnKindDef[@Name="LionMilBase"]/inventoryOptions/subOptionsChooseOne/li[thingDef="Silver"]/countRange/max</xpath>
+				<value>
+					<max>1500</max>
```

Comment:
Honestly, unless there's a really compelling reason not to, I wouldn't let them carry more than 100 - 200 at most. Not only is there the inventory considerations as you mentioned, but the wealth bloat for any sizable raid with raiders carrying several hundred silver would be considerable.

Reply:
Agreed!

Fixed in c0dbd18.